Bottom: c167165d21a439910b715620b3642f6f149d7666
Top:    be3c97a1928367b99d328a8467e1472f798770ff
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:50:19 +0000

Refresh of add-reversed-orientation-field

---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index c59680d..95232a6 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,6 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
-  bool plane_orientation_reversed;
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
